components/TextInput.js

```diff
@@ -27,7 +27,13 @@ class TextInput extends PureComponent {
   }
 
   render() {
-    const { errorMessage, highlightOnFocus, style, ...otherProps } = this.props;
+    const {
+      errorMessage,
+      highlightOnFocus,
+      startErrorAnimation,
```

Comment:
i don't like the new prop for this. It should be tied to the errorMessage prop somehow